docs/layouts/shortcodes/generated/execution_configuration.html

```diff
@@ -8,6 +8,31 @@
         </tr>
     </thead>
     <tbody>
+        <tr>
+            <td><h5>execution.async-mode.buffer-size</h5></td>
```

Comment:
Given that these configurations are only for internal experimental use, it might be better not to expose them in documents.